tests/python/requirements/PySafetyBearTest.py

```diff
@@ -60,3 +60,17 @@ def test_with_no_pinned_requirements(self):
         ) as check:
             self.check_validity(self.uut, ['foo', 'bar>2'])
             assert not check.called
+
+    def test_with_key_advisory(self):
```

Comment:
This test looks too similar to `test_with_cve_vulnerability` test above. Do we really need this test?

Reply:
I think No,as data we are getting is already  a updated data in the form of json

Reply:
I will make changes

Reply:
Done